packages/api/amplify_api_dart/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: amplify_api_dart
 description: The Amplify API category plugin in Dart-only, supporting GraphQL and REST operations.
-version: 0.1.0
+version: 0.1.0+1
```

Comment:
nit: we haven't released this package yet so I don't believe a bump is needed in this package, but no harm having it I suppose.